fdbserver/TenantCache.actor.cpp

```diff
@@ -51,15 +51,15 @@ class TenantCacheImpl {
 	ACTOR static Future<Void> build(TenantCache* tenantCache) {
 		state Transaction tr(tenantCache->dbcx());
 
-		TraceEvent(SevInfo, "BuildingTenantCache", tenantCache->id()).log();
+		TraceEvent(SevInfo, "TenantCacheBuildStarting", tenantCache->id());
```

Comment:
an alternate would be to introduce a KNOB to control the logging for TenantCache, that ways we might have control better control. Thoughts?

Reply:
That's a good idea. We could even have a bunch of the SevDebug's be SevInfo's instead if the verbose logging is turned on. I'll plan to do that for the next release.